release-notes/azure/v11.2.2.md

```diff
@@ -0,0 +1,12 @@
+⚡️ Giant Swarm Release 11.2.2 for Azure ⚡️
+
+This release contains improvements in upgrade procedure.
+We have improved new nodes discovery at the stage, where the cluster size is doubled with new versions of the machines.
+This will make sure that in case of upgrade disruptions, process will follow its original path. 
+
+To upgrade, please contact your Solution Engineer.
+
+Below, you can find more details on components that were changed with this release.
```

Comment:
```suggestion
This release contains improvements in the upgrade procedure.
We have improved new nodes discovery at the stage where the cluster size is doubled with new versions of the machines.
This will make sure that in case of upgrade disruptions, the process will follow its original path. 

To upgrade, please contact your Solution Engineer.

Below, you can find more details on components that were changed with this release.
```